docs/sources/k6/next/javascript-api/k6-browser/metricmessage.md

```diff
@@ -0,0 +1,60 @@
+---
+title: 'MetricMessage'
+slug: 'metricmessage'
+description: 'Browser module: MetricMessage Object'
+weight: 03
+---
+
+# MetricMessage
+
+A `MetricMessage` object allows tagging of metrics that are measured and emitted for the page.
+
+`MetricMessage` objects are dispatched by the page when a handler is registered with [page.on('metric')](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-browser/page/on). For each metric that is measured and emitted for the page, the k6 browser delivers a `MetricMessage` object to the registered handlers, allowing them to pattern match and tag metrics.
+
+## tag
+
+The `tag` method matches the given `matches` with the current metric's url and name tags. When a match is found, it will use `name` to replace the existing url and name tag values.
+
+Doing this helps to group metrics with disparate url and name tags, which are, in fact, referencing the same resource so that a correlation can be found over time and to reduce the cardinality of the metrics.
```

Comment:
```suggestion
Doing this helps group metrics with different URL and name tags, which are, in fact, referencing the same resource so that a correlation can be found over time and to reduce the cardinality of the metrics.
```